src/tests/ftest/util/dfuse_utils.py

```diff
@@ -392,13 +392,55 @@ def get_stats(self):
         cmd = f"daos filesystem query --json {self.mount_dir.value}"
         result = run_remote(self.log, self.hosts, cmd)
         if not result.passed:
-            raise CommandFailure(f'"fs query failed on {result.failed_hosts}')
+            raise CommandFailure(f'fs query failed on {result.failed_hosts}')
 
         data = json.loads("\n".join(result.output[0].stdout))
         if data["status"] != 0 or data["error"] is not None:
             raise CommandFailure("fs query returned bad data.")
         return data["response"]
 
+    def get_log_file(self):
```

Comment:
To avoid confusion, maybe `get_log_file_data()` be clearer.  We have other classes with `get_log_file()` methods that just return the name of the log file.

Reply:
- [ ] Refactor function name get_log_file() to get_log_file_data()

Reply:
> * [x]  Refactor function name get_log_file() to get_log_file_data()

Fixed with commit 2e9afbd28